src/content/en/2020/accessibility.md

```diff
@@ -236,7 +236,7 @@ There are 5 rules of ARIA that we need to understand before  making use of this
 
 ##### The 5 Rules of ARIA (from [W3C’s Using Aria](https://www.w3.org/TR/using-aria/)):
 
-1. If you can use a native HTML element [HTML 5.1] or attribute with the semantics and behavior you require already built in, instead of repurposing an element and adding an ARIA role, state or property to make it accessible, then do so.
+1. If you can use a native HTML element [HTML5.1] or attribute with the semantics and behavior you require already built in, instead of repurposing an element and adding an ARIA role, state or property to make it accessible, then do so.
```

Comment:
Could you revert the spacing changes for "HTML"? It uses a space in the spec: https://www.w3.org/TR/html51/ 
```suggestion
1. If you can use a native HTML element [HTML 5.1] or attribute with the semantics and behavior you require already built in, instead of repurposing an element and adding an ARIA role, state or property to make it accessible, then do so.
```